File: api/code-samples/python/debug/profiling/blockProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "block.profile"
seconds = 10

sdk = OpenSDK(host)
debug_response = sdk.debug.block_profile(fileName, seconds)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/blockProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "block.profile";
    const seconds = 10;

    sdk.debug.blockProfile(file, seconds, {}, (err, data, response) => {
        console.log(data);
    });
}
)()